integrated_channels/sap_success_factors/client.py

```diff
@@ -73,6 +73,13 @@ def get_oauth_access_token(url_base, client_id, client_secret, company_id, user_
             return data['access_token'], datetime.datetime.utcfromtimestamp(data['expires_in'] + int(time.time()))
         except (KeyError, ValueError) as error:
             raise ClientError(response, response.status_code) from error
+        except TypeError as error:
+            LOGGER.error(
+                'SAP SF OAuth2 POST response is of invalid format. Error: [%s], Response: [%s]',
```

Comment:
Should we add user_id or company_id in the logger message or not?
That might help in the future for searching purposes.
@irfanuddinahmad

Reply:
Done. Good catch!